plugins/item_previews/web_client/stylesheets/itemPreviewWidget.styl

```diff
@@ -16,33 +11,28 @@ $maxPreviewHeight = 400px
   padding 5px 6px
 
 .g-widget-item-previews-container
-  max-height $maxPreviewHeight
+  max-height 400px
   overflow-x auto
   white-space nowrap
   &.g-wrap
     white-space normal
 
 .g-widget-item-preview
-  display inline
-  margin-right 10px
+  margin 0 10px 10px 10px
+  padding-bottom 5px
+  border-bottom 1px solid #eee
 
 .g-widget-item-previews-container img
   max-height $maxPreviewHeight
   border-radius 5px
   width auto
 
-.g-widget-item-previews-container .panel
-  display inline-block
-  vertical-align top
-  width 30% !important
-
-.g-widget-item-previews-container .panel .panel-body
-  padding 5px
-
 .g-widget-item-previews-container .json
   background-color #f5f5f5
   border-radius 4px
   border 1px solid #ccc
   color #333
-  height 300px
+  height $maxPreviewHeight
+  width 30%
```

Comment:
Yea, it's kind of too small. The reason I didn't make it 100% is that a scrollable textarea will make it harder to scroll the container. (The user will scrolll the textarea instead of the div because the mouse cursor is on the textrea). I changed it to a `calc(100% - 100px)` to leave some space for scrolling at both ends.